Userland/Libraries/LibGUI/TabWidget.h

```diff
@@ -12,15 +12,17 @@
 
 namespace GUI {
 
+enum class TabPosition {
+    Top,
+    Bottom,
+    Left,
+    Right,
+};
+
 class TabWidget : public Widget {
     C_OBJECT(TabWidget)
 public:
-    enum TabPosition {
-        Top,
-        Bottom,
-        Left,
-        Right,
-    };
+    typedef GUI::TabPosition TabPosition;
```

Comment:
We use `using` instead of `typedef`. But why not just update the references to this enum to remove the cruft? There's only a handful of references.